editor/editor_node.cpp

```diff
@@ -6502,13 +6506,16 @@ static Node *_resource_get_edited_scene() {
 }
 
 void EditorNode::_print_handler(void *p_this, const String &p_string, bool p_error, bool p_rich) {
```

Comment:
It's no longer using `p_this`. What was it for in the first place since we have singleton access?

Reply:
I think the prototype of print handler function was designed similarly to thread functions, in that they get an arbitrary user data pointer that the caller code can use for anything. There are some resource loader error callbacks I stripped their counterpart parameter from because no known use case needed it. However, in this case, I've decided to keep it since I thought in this case the "users" could be of very varied natures. That said, I didn't check if others actually need the parameter and we could consider a similar surgery if none do.